src/connectors/breadcrumb/connectBreadcrumb.ts

```diff
@@ -186,6 +175,38 @@ const connectBreadcrumb: BreadcrumbConnector = function connectBreadcrumb(
         unmountFn();
       },
 
+      getWidgetRenderState(renderState, { results, state }) {
```

Comment:
@yannickcr Bouncing back on https://github.com/algolia/instantsearch.js/pull/4457#discussion_r461592742 — this is why it's convenient to be explicit about `results: undefined` in TypeScript, otherwise we wouldn't be able to use `results` here.